lib/src/main/java/graphql/nadel/validation/NadelTypeValidation.kt

```diff
@@ -146,67 +146,48 @@ internal class NadelTypeValidation {
         )
     }
 
-    /**
-     * Answers whether `rhs` assignable to `lhs`?
-     *
-     * i.e. does the following compile
-     *
-     * ```
-     * vol output: lhs = rhs
-     * ```
-     *
-     * Note: this assumes both types are from the same schema. This does NOT
-     * deal with differences between overall and underlying schema.
-     */
-    context(NadelValidationContext)
-    fun isAssignableTo(lhs: GraphQLNamedOutputType, rhs: GraphQLNamedOutputType): Boolean {
```

Comment:
Moved this function to hydration validation. Nobody else was using it anyway.

It was causing a circular dependency.